src/lib.rs

```diff
@@ -509,3 +520,26 @@ impl DerInner<'_> {
         })
     }
 }
+
+// Format an iterator of u8 into a hex string
+fn hex<'a>(f: &mut fmt::Formatter<'_>, payload: impl IntoIterator<Item = &'a u8>) -> fmt::Result {
+    for b in payload {
+        write!(f, "{:02x}", b)?;
+    }
+    Ok(())
+}
+
+#[cfg(feature = "std")]
+mod tests {
+    #[test]
+    fn der_debug() {
+        let der = super::Der::from_slice(&[0x01, 0x02, 0x03]);
+        assert_eq!(format!("{:?}", der), "010203");
+    }
+
+    #[test]
+    fn algid_debug() {
```

Comment:
Nit: I think `alg_id` would be a more idiomatic spelling.

Reply:
Fixed.